kura/org.eclipse.kura.driver.ble.sensortag.provider/src/main/java/org/eclipse/kura/internal/driver/ble/sensortag/TiSensorTagGatt.java

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Eurotech and/or its affiliates
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.internal.driver.ble.sensortag;
+
+import java.util.UUID;
+
+public class TiSensorTagGatt {
+
```

Comment:
would be good if you can provide a private constructor

Reply:
👍